packages/nvidia-container-toolkit/nvidia-container-toolkit-config-k8s

```diff
@@ -0,0 +1,12 @@
+[required-extensions]
+kubernetes = "v1"
+
++++
+accept-nvidia-visible-devices-as-volume-mounts = {{settings.kubernetes.nvidia.container-runtime.visible-devices-as-volume-mounts}}
+accept-nvidia-visible-devices-envvar-when-unprivileged = {{settings.kubernetes.nvidia.container-runtime.visible-devices-envvar-when-unprivileged}}
```

Comment:
Let's be safe, and use the `{{default}}` helper, otherwise if `settings.kubernetes.nvidia.container-runtime.visible-devices-as-volume-mounts` isn't present, the render will fail.